doc/user/content/get-started/install-materialize-emulator.md

```diff
@@ -62,13 +62,13 @@ not suitable for production deployments</redb>.
    been already downloaded.
 
    ```sh
-   docker run -v mzdata:/mzdata -p 6875:6875 materialize/materialized
+   docker run -d -p 6875:6875 -p 6876:6876 materialize/materialized:latest
```

Comment:
```suggestion
   docker run -d -p 6875:6875 -p 6876:6876 materialize/materialized:{{< version >}}
```

This will avoid the issue with `latest` being stale that you ran into!